pkg/controller/storagecluster/reconcile.go

```diff
@@ -593,16 +593,90 @@ func newCleanupJob(sc *ocsv1.StorageCluster) *batchv1.Job {
 				Spec: corev1.PodSpec{
 					RestartPolicy:      corev1.RestartPolicyNever,
 					ServiceAccountName: "rook-ceph-system",
+					Volumes: []corev1.Volume{
+						{
+							Name:         "ceph-conf-emptydir",
+							VolumeSource: corev1.VolumeSource{EmptyDir: &corev1.EmptyDirVolumeSource{}},
+						},
+						{
+							Name:         "rook-config",
+							VolumeSource: corev1.VolumeSource{EmptyDir: &corev1.EmptyDirVolumeSource{}},
+						},
+					},
+
 					Containers: []corev1.Container{
 						{
 							Name:  "operator",
 							Image: os.Getenv("ROOK_CEPH_IMAGE"),
-							Command: []string{
+							Args: []string{
```

Comment:
Why are you switching to `Args` instead of `Command`? Shouldn't you ideally be using both?

Reply:
By not specifying the `Command`, it will use the default container entrypoint, which is the same pattern used by the rook operator.

Reply:
When the job was triggered using `command`, it was failing to run the ceph commands with this error message- `Error initializing cluster client: ObjectNotFound('RADOS object not found (error calling conf_read_file)',)`.